pkg/auth/authenticator/password/keystonepassword/keystonepassword.go

```diff
@@ -0,0 +1,66 @@
+package keystonepassword
+
+import (
+	"fmt"
+	"errors"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack"
```

Comment:
> need to add the godeps for this... how big are those?

When you add them, see the commit format.  We keep the godeps in different commits starting with `bump(dependency name): commit_id`